sycl/include/CL/sycl/aliases.hpp

```diff
@@ -80,7 +80,8 @@ using ulong = unsigned long;
 using longlong = long long;
 using ulonglong = unsigned long long;
 // TODO cl::sycl::half is not in SYCL specification, but is used by Khronos CTS.
-using half = half;
+// Related tests issue: KhronosGroup/SYCL-CTS#37
+using half = cl::sycl::detail::half_impl::half;
```

Comment:
This file declares a user-visible aliases, and for now I would like to have both of them (until the SYCL spec is not clarified, see KhronosGroup/SYCL-Docs#40).

In other files, these usings are just forward-declarations to avoid extra `#include` directives